src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
 // write your code here
+const populationArray = document.querySelectorAll('.population');
+const totalPopulationElement = document.querySelector('.total-population');
+const averagePopulationElement = document.querySelector('.average-population');
+let totalPopulation = 0;
+
+populationArray.forEach((item) => {
+  const population = +item.textContent.replaceAll(',', '');
```

Comment:
The use of `replaceAll` might not be supported in all environments. Consider using `replace` with a global regular expression if compatibility is a concern.